### PR TITLE
Fixed #20

### LINKED ISSUE #20: Get `/user/:user_id/types/:type_id` api
YAML definition is here:
https://github.com/Greenstand/treetracker-like/blob/61faba07eb6d5509a77f91a1d865519eb5a99575/docs/api/spec/query-api.yaml#L215

### PR DESCRIPTION
**Pull request by Shpendiiiii**
Dear Reviewer,

Please provide your .env file with the database url of your postgre instance.

Then: 

- ```prisma db push``` to apply migrations to the db
- ```yarn prisma:seed``` to insert the test data to the db
- ```nest start like``` to start the `like` app, since it is the default one, simply running ```nest start``` will work as well.

Please go to /docs to access the swagger ui. 

Best, 

Shpendi

### CODE REVIEW

**Review by Mohmn — APPROVED** [review]

**Comment by Mohmn** [comment]
i havent run these changes on local but they look good to me, once i resolve the baseline migation issue on my local
i will throughly review it once again , but for now changes look good to me @Shpendiiiii @dadiorchen @GuningShen

File: apps/like/src/object/object.controller.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+  Controller,
+  Get,
+  Post,
+  Put,
+  ValidationPipe,
+  HttpStatus,
+  Param,
+  Req,
+  Res,
+} from '@nestjs/common';
```
Comment (dadiorchen) [inline apps/like/src/object/object.controller.ts:12]: Maybe we should add this: https://github.com/Greenstand/treetracker-like/issues/34

File: apps/like/src/main.ts [hunk 2]
```diff
@@ -1,14 +1,16 @@
 import { Logger } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { setupSwagger } from '../../../libs/utils/swagger';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   const port = process.env.PORT || 3006;
+  if (process.env.NODE_ENV !== 'production') {
+    setupSwagger(app);
```
Comment (dadiorchen) [inline apps/like/src/main.ts:10]: Very useful for development and coorditation between devs, thank you!

**Review by dadiorchen — APPROVED** [review]

File: libs/validation/src/dto/create-post.dto.ts
Comment (GuningShen) [inline libs/validation/src/dto/create-post.dto.ts:1]: This file is irrelavant (remove in the future)

**Review by GuningShen — APPROVED** [review]
All changes look good to me! Thanks for your contribution. You could remove irrelevant DTO files in the future.

**Pull request merged**